linux_pipeline/Jenkinsfile_rhel_kernel_validation

```diff
@@ -99,3 +107,30 @@ stage ("Test kernel") {
         parallel runs
     }
 }
+
+stage ("Send Report"){
```

Comment:
I would like another PR where all the stage names are in the format send_email_report